<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TerminalNum</class>
 <widget class="QMainWindow" name="TerminalNum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Terminal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Terminal Number:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TAddBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TerminalTable">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Terminal ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Terminal Number</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="DeleteBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Terminal Details</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TBackBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TerminalNum_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete Terminal Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DelTerminalNum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
